src/serializers/meal-plans-serializer.js

```diff
@@ -11,6 +11,19 @@ const planResourceType = planResourceProp.type.enum[0];
 const planResourceAttributes = planResourceProp.attributes.properties;
 const planResourceKeys = _.keys(planResourceAttributes);
 
+/**
+ * Prepares raw data for serializer
+ *
+ * @param {object[]} rawMealPlans raw meal plan data
+ */
```

Comment:
What should be returned?

Reply:
The function doesn't return anything. Do you think it should?

Reply:
Oh, I didn't notice. Feel free to ignore my comment here.